ODATA-1464 (#70)
oasis-tcs/odata-specs: odata-csdl/14 Vocabulary and Annotation.md

#### ##subsubsubsec Function `odata.matchesPattern`

The `odata.matchesPattern` client-side function takes two string

> expressions as arguments and returns a Boolean value. The existence of this function
> follows from the URL function of the same name [OData-URL, section 5.1.1.7.1](#ODataURL).

Reviewer: ```suggestion
expressions as arguments and returns a Boolean value.
It is the counterpart of the identically named URL function [OData-URL, section 5.1.1.7.1](#ODataURL).
```
